tests/cypress/integration/actions_users/registration_involved/case_4_assign_taks_job_users.js

```diff
@@ -77,7 +80,7 @@ context('Multiple users. Assign task, job. Deactivating users.', () => {
                 secondUser.password,
             );
             cy.createAnnotationTask(
-                taskName,
+                `${taskName} second`,
```

Comment:
Maybe create another identificator to store the second task name?

Reply:
Done.